site/content/entry/citra-progress-report-2017-q4.md

```diff
@@ -0,0 +1,202 @@
++++
+date = "2018-02-03T12:07:00-04:00"
+title = "Citra Progress Report - 2017 Q4"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 9777
++++
+
+New year, new changes in Citra. Specifically, lots of technical changes under
+the hood, from applets to IPC, have made Citra even more accurate and laid a
+foundation for even more goodies the next year. With many great changes all
+around, this article is going to be packed to the brim with all the new goodies
+that have come to Citra during those crimson months.
+
+But, enough faffing about! Let's get right into it:
+
+## [citra-qt : Adding fullscreen mode](https://github.com/citra-emu/citra/pull/3001) by [Styleoshin](https://github.com/Styleoshin)
+
+One of the most requested features for Citra, second only to controller support,
```

Comment:
leave out "second only to controller support" the number one request is speed ;)